components/script/dom/bindings/root.rs

```diff
@@ -309,20 +309,20 @@ impl<'root, T: RootedReference<'root> + 'root> RootedReference<'root> for Option
 /// on the stack, the `Dom<T>` can point to freed memory.
 ///
 /// This should only be used as a field in other DOM objects.
-#[must_root]
-pub struct Dom<T> {
+// #[must_root]
```

Comment:
Why is this commented out? It's pretty important for the safety of `Dom<T>`.

Reply:
To be honest, I don't remember. Maybe I thought that making T a `#[must_root]` is enough, but then the compiler would skip the pointer reference. Maybe I commented it out, because it produced a lot of errors that I couldn't get rid of. If it is the case, then it was probably before I started to put items in temporary exception list.